src/lib/cosmos/connectors/network.ts

```diff
@@ -0,0 +1,19 @@
+import { initializeConnector } from '@web3-react/core'
+import { Network } from '@web3-react/network'
+import { SupportedChainId } from 'constants/chains'
+
+export const URLS = {
```

Comment:
Can we re-use our existing URLs list?

Reply:
No, since that relies on getting the infura key from `process.env.REACT_APP_INFURA_KEY` (and CRA values aren't available in cosmos)